frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/AboutTab/CreatedFor/CreatedFor.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode } from 'react';
+import classes from './CreatedFor.module.css';
+import { Label, Paragraph } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+import { RepositoryType } from 'app-shared/types/global';
+import { Repository } from 'app-shared/types/Repository';
+import { PersonCircleIcon } from '@navikt/aksel-icons';
+import { formatDateToDateAndTimeString } from 'app-development/utils/dateUtils';
+
+export type CreatedForProps = {
+  /**
+   * The repository type
+   */
+  repositoryType: RepositoryType;
+  /**
+   * The repository
+   */
+  repository: Repository;
+  /**
+   * The name of the author
+   */
+  authorName: string;
+};
+
+/**
+ * @component
+ *    Displays the section for created for in the About Tab in the Settings modal
```

Comment:
Det er fint med dokumentasjon, men det er viktig å passe på at den ikke kan misforstås - da virker den bare mot sin hensikt. Denne setningen tok det litt tid å forstå før jeg hadde lest navnet på komponenten.
```suggestion
 *    Displays the section for "created for" in the About Tab in the Settings modal
```